Header header
uint8 BotState
uint8 STATIONARY= 0
uint8 MOVING= 1
float32 x
float32 y 